<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Translate</class>
 <widget class="QDialog" name="Translate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Translate</string>
  </property>
  <widget class="QPushButton" name="pBGenarateTranlateFile">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;To genarate Google tranlation input file click here&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Genarate </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBUpdateTsFile">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;To update .ts file with tranlating click here&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Update .ts File</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>260</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select updating ts file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEupdatingTsFile">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBselectUpdatingTsFile">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select ts file that need to update with tranlation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select Translating .ts file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lETranlatingFilePath">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBSelectTranalatingFile">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select qt tranlation ts file that need to translate&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Select Google Tranlated file</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEGoogleTranlatedFile">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pBSelectGoogleTranlatedFile">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select Google tranlator tranlated file &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>56</y>
     <width>521</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Google tranlating file save path</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lEgoogletranlatedfileSavePath">
      <property name="minimumSize">
       <size>
        <width>170</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select Directry to save google tranlating input file&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Select dir</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>504</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBSelectTranalatingFile</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>selectTranslatingFileClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>498</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBSelectGoogleTranlatedFile</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>selectGoogleTranslatedFileClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>499</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBGenarateTranlateFile</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>genarateGoogleTranlatingInputFileClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBUpdateTsFile</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>genarateUpdateTsFileClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>195</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>selectGoogleTranslatedFileSavePathClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>522</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>763</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBselectUpdatingTsFile</sender>
   <signal>clicked()</signal>
   <receiver>Translate</receiver>
   <slot>selectUpdatingTsFileClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>757</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectTranslatingFileClicked()</slot>
  <slot>selectGoogleTranslatedFileClicked()</slot>
  <slot>genarateGoogleTranlatingInputFileClicked()</slot>
  <slot>genarateUpdateTsFileClicked()</slot>
  <slot>selectGoogleTranslatedFileSavePathClicked()</slot>
  <slot>selectUpdatingTsFileClicked()</slot>
 </slots>
</ui>
